### DISCUSSION TITLE
Enable IE11 Support for version 7

### DISCUSSION THREAD

**Discussion by theboyWhoCriedWoolf** [react-hook-form/react-hook-form#5370] [General]
Hello,

I am trying to use V7 for IE11 still as for the next few months we will be supporting it within our project. I however want to use V7 so we can future proof our implementations of hooks form. 

Is there a way this can be achieved? What polyfills etc are required that might have been removed for V7?

Any support or help would be much appreciated.

**Comment by bluebill1049** [comment]
Please stay with v6, v7 will no longer support IE 11.
